R/estimate_cfr.R

```diff
@@ -0,0 +1,8 @@
+# Function to calculate cfr by specifying columns from a dataframe with
+# deaths and cases
+estimate_cfr <- function(df, numerator, denominator) {
```

Comment:
Can the {cfr} package not calculate this when a delay distribution is not supplied?

Reply:
yes it can, initially I wasn't using it for 2 reasons, 1/ because the output includes the 95 CI and I just wanted the CFR value, so I thought it would be easier to write a function that is doing exactly what I want it to do, 2/ because I wanted the package to be used just for the comparison of the methods, but I guess I could still use it anyway if a delay distribution is not being used... do you think it would be better?